src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/PublishTaxonomyFunction.cs

```diff
@@ -31,9 +29,7 @@ public PublishTaxonomyFunction(IContentService contentService)
                 executionContext.FunctionName,
                 message.ToString());
 
-            var context = new PublishContext(DateTime.UtcNow, false);
-
-            await _contentService.UpdateTaxonomy(context);
+            await _contentService.DeleteAllBlobCacheItems();
```

Comment:
There's other stuff cached, so I think we should come up with a better name for this, since we are not deleting all blob cache items which is a bit misleading.

Reply:
After running it past a couple of people, gone with `DeleteCachedTaxonomyBlobs`.